apps/site/pages/en/learn/modules/package-configuration.md

```diff
@@ -0,0 +1,563 @@
+---
+title: Package Configuration
+layout: learn
+authors: JakobJingleheimer
```

Comment:
```suggestion
authors: JakobJingleheimer, joyeecheung
```

@joyeecheung you've contributed quite significantly to this. Shall I add you as one of the authors?

Reply:
I don't feel comfortable adding my name to a document full of inappropriate cocktail jokes TBH.